lib/active_admin/views/components/status_tag.rb

```diff
@@ -50,7 +47,7 @@ def build(*args)
         super(content, options)
 
         add_class(status_to_class(status)) if status
-        add_class(type.to_s) if type
+        add_class(type.to_s) if type && !type.is_a?(Hash)
```

Comment:
Yes it will be better and less work. Cucumber tests are passing with it going back to being a splat. 👊🏻